src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -743,4 +743,11 @@
   <data name="BadBuiltInOps3" xml:space="preserve">
     <value>'{0}' was not a valid built-in operator name</value>
   </data>
+  <data name="HintNameInvalidSegment" xml:space="preserve">
+    <value>The hintName '{0}' contains an invalid segment '{1}' at position {2}.</value>
+    <comment>{0}: the provided hintname. {1}: the invalid segment, {2} the position it occurred at</comment>
+  </data>
+  <data name="HintNameNotWithinCurrentDirectory" xml:space="preserve">
+    <value>The hintName '{0}' is not within the current directory.</value>
```

Comment:
I expected to see this message appear in some test, but couldn't find it.

Reply:
Yes, we only tested whether an `ArgumentException` is thrown and whether it contains the `hintName` in it; not the exception message. Should I extend the tests to do that? And since the message is localized, should I use resources as the expected values?

Reply:
Making sure the code was exercised is probably sufficient. Thanks